Sources/Steepshot/Steepshot.iOS/Cells/PhotoGalleryCell.cs

```diff
@@ -24,5 +25,10 @@ public void UpdateImage(UIImage image)
         {
             _bodyImage.Image = image;
         }
+
+        public void UpdateImage(string url)
+        {
+            ImageService.Instance.LoadUrl(url).Into(_bodyImage);
```

Comment:
We should not load full image in small cell (See photocollectionviewcell for example)